tests/models/test_model_config.py

```diff
@@ -13,7 +13,7 @@
 
 def test_config_not_set():
     with pytest.raises(
-        FileNotFoundError, match="Config file is None. Please provide a valid path."
+        FileNotFoundError, match="Config file is not provided which is needed to load the model."
     ):
         ModelConfig()
 
```

Comment:
Can we add a test here where the global is set to "", development_config is set to a valid config path, and it fails?